README.md

```diff
@@ -114,6 +114,57 @@ await fastify.ready()
 console.log(fastify.secrets.db.PG_PASS)
 ```
 
+#### Refreshing Secrets
+
+In the event secrets need to be dynamically refreshed, a refresh method is exposed to allow for the refreshing of single, sets, or all secrets scoped to the provided namespace. The signature of the refresh method is as follows,
```

Comment:
```suggestion
In the event secrets need to be dynamically refreshed, a refresh method is exposed to allow for the refreshing of single, sets, or all secrets scoped to the provided namespace. The signature of the refresh method is as follows:
```

Reply:
also, let's make extra sure that this all works with both the array mode and object mode definition of secrets please!

Reply:
Added an explicit test to cover this scenario [here](https://github.com/nearform/fastify-secrets-core/pull/173/files#diff-af83e5a832d61d0b6ce097e2004c1d7e9745c3b2d797cc3ed6e16c5790146108R473).